sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/main/java/com/azure/messaging/eventhubs/checkpointstore/blob/BlobPartitionManager.java

```diff
@@ -90,57 +92,48 @@ public BlobPartitionManager(ContainerAsyncClient containerAsyncClient) {
-                String blobName = getBlobName(partitionOwnership.getEventHubName(),
-                    partitionOwnership.getConsumerGroupName(), partitionId);
-
-                if (!blobClients.containsKey(blobName)) {
-                    blobClients.put(blobName, containerAsyncClient.getBlobAsyncClient(blobName));
-                }
-
-                BlobAsyncClient blobAsyncClient = blobClients.get(blobName);
-
-                Metadata metadata = new Metadata();
-                metadata.put(OWNER_ID, partitionOwnership.getOwnerId());
-                Long offset = partitionOwnership.getOffset();
-                metadata.put(OFFSET, offset == null ? null : String.valueOf(offset));
-                Long sequenceNumber = partitionOwnership.getSequenceNumber();
-                metadata.put(SEQUENCE_NUMBER, sequenceNumber == null ? null : String.valueOf(sequenceNumber));
-                BlobAccessConditions blobAccessConditions = new BlobAccessConditions();
-                if (ImplUtils.isNullOrEmpty(partitionOwnership.getETag())) {
-                    // New blob should be created
-                    blobAccessConditions.setModifiedAccessConditions(new ModifiedAccessConditions()
-                        .setIfNoneMatch("*"));
-                    return blobAsyncClient.asBlockBlobAsyncClient()
-                        .uploadWithResponse(Flux.just(UPLOAD_DATA), 0, null, metadata, null,
-                            blobAccessConditions)
-                        .flatMapMany(response -> {
-                            partitionOwnership.setETag(response.getHeaders().get(ETAG).getValue());
-                            return Mono.just(partitionOwnership);
-                        }, error -> {
-                                logger.info("Couldn't claim ownership of partition {}, error {}", partitionId,
-                                    error.getMessage());
-                                return Mono.empty();
-                            }, Mono::empty);
-                } else {
-                    // update existing blob
-                    blobAccessConditions.setModifiedAccessConditions(new ModifiedAccessConditions()
-                        .setIfMatch(partitionOwnership.getETag()));
-                    return blobAsyncClient.setMetadataWithResponse(metadata, blobAccessConditions)
-                        .flatMapMany(response -> {
-                            partitionOwnership.setETag(response.getHeaders().get(ETAG).getValue());
-                            return Mono.just(partitionOwnership);
-                        }, error -> {
-                                logger.info("Couldn't claim ownership of partition {}, error {}", partitionId,
-                                    error.getMessage());
-                                return Mono.empty();
-                            }, () -> Mono.empty());
-                }
+        return Flux.fromArray(requestedPartitionOwnerships).flatMap(partitionOwnership -> {
```

Comment:
In general, I'd limit formatting changes to a separate PR. I have to do a line by line comparison to see what actually changed and it makes it harder to see the functional changes you made.

Reply:
Yeah, I'd rather have not done these changes but Checkstyle's was complaining a lot about indenting on this file.